Sources/App/Core/Search.swift

```diff
@@ -390,6 +390,8 @@ enum Search {
         return query.all(decoding: DBRecord.self)
             .mapEachCompact(Result.init)
             .map { results in
+                SearchLogger.log(query: sanitizedTerms.joined(separator: " "), results: results)
```

Comment:
```suggestion
                SearchLogger.log(query: sanitizedTerms.joined(separator: " "), results: results.prefix(10))
```

I still need to check how many searches we're getting but let's definitely limit this to 10 for starters and perhaps `pageSize` (20) later.